src/DataSource/BaseDataSource.tsx

```diff
@@ -191,15 +190,14 @@ export default abstract class BaseDataSource extends React.Component<
 
   public render() {
     const WrappedComponet = this.getWrappedComponent();
-
     return (
       <DataSourceContext.Provider
         value={{
           actions: this.getActions(),
           state: { ...this.state }
         }}
       >
-        <WrappedComponet error={this.state.error} {...this.props} />
+        <WrappedComponet {...{error:this.state.error, refresh:() => this.retrieveData.bind(this) }}  />
```

Comment:
But if the component has props.children?

Reply:
the props.children is always present in react

Reply:
👍